src/main/java/core/basesyntax/service/impl/ProccessDataImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProccessData;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class ProccessDataImpl implements ProccessData {
+
+    private static final OperationStrategyImpl operationStrategyImpl = new OperationStrategyImpl();
```

Comment:
use abstraction over implementation 
it shouldn't be static 
you should set it via constructor